src/main/kotlin/no/nb/mlt/wls/infrastructure/email/DisabledEmailNotifier.kt

```diff
@@ -7,19 +7,19 @@ import no.nb.mlt.wls.domain.ports.outbound.UserNotifier
 
 private val logger = KotlinLogging.logger {}
 
+const val MESSAGE = "Sending emails for orders is disabled"
+
 class DisabledEmailNotifier : UserNotifier {
-    override suspend fun orderConfirmation(order: Order): Boolean {
-        logger.warn { "Sending emails for orders is disabled" }
-        return true
-    }
+    override suspend fun orderConfirmation(order: Order) = logOrderDisabled()
 
-    override suspend fun orderPickup(orderPickupData: OrderPickupMail.OrderPickupData): Boolean {
-        logger.warn { "Sending emails for orders is disabled" }
-        return true
-    }
+    override suspend fun orderPickup(orderPickupData: OrderPickupMail.OrderPickupData) = logOrderDisabled()
+
+    override suspend fun orderCompleted(order: Order) = logOrderDisabled()
```

Comment:
Maybe add the return type for these functions too, so it is consistent with other notifier?

```suggestion
    override suspend fun orderConfirmation(order: Order): Boolean = logOrderDisabled()

    override suspend fun orderPickup(orderPickupData: OrderPickupMail.OrderPickupData): Boolean = logOrderDisabled()

    override suspend fun orderCompleted(order: Order): Boolean = logOrderDisabled()
```